### PR TITLE
Weather APP Sthlm

### PR DESCRIPTION
**Pull request by elol42**
Weahter APP. I wanted a background under the header but i didn´t get it to fit my raduis on the header.. Do you have any tips for me that i could try?

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Hi, can you please add your Netlify link to the PR?

**Comment by elol42** [comment]
Netlify Link:
https://distracted-darwin-214f59.netlify.com/

File: script.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    else if (json.weather[0].description.includes("snow")) {
+      document.getElementById("weatherIcon").src = "snow.png"
+    }
+    else {
+      document.getElementById("weatherIcon").src = "cloudysun.png"
+    }
+  })
+
+
+fetch(`https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=d1862226e2bf07d0d8d418921e586230`)
```
Comment (JennieDalgren) [inline script.js]: I'm not really sure why you're doing another fetch here? You get the same response, right?
If that's the case, stick to one fetch and do the sunrise/sunset things in the same place ☝🏼

File: style.css [hunk 2]
```diff
@@ -0,0 +1,224 @@
+body {
+  font-family: 'Roboto', sans-serif;
+  font-weight: lighter;
+  margin: -0px;
+}
+
+header {
+  background-image: linear-gradient( rgba(214, 132, 226, 0.795), rgba(113, 165, 233, 0.795));
```
Comment (JennieDalgren) [inline style.css]: fancy with a gradient! 😎

File: style.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+.bigTemp {
+  margin-bottom: -65px;
+  margin-top: -100px;
+}
+
+.cityTop {
+  margin-bottom: -43px;
+}
+
+.weatherdescription {
```
Comment (JennieDalgren) [inline style.css]: remember code guide lines about naming classes.
`weather-description`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+}
+
+.weatherforecast{  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.forecastHead{
```
Comment (JennieDalgren) [inline style.css]: remember code guide lines about naming classes.
`forecast-head`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+  font-size: 38px;
+}
+
+  .hamburger {
+    display: none;
+  }
+
+  .topweather {
+    margin-left: 17%;
+  }
```
Comment (JennieDalgren) [inline style.css]: already specified in "main" styling

**Pull request closed without merging**